Formula/ceres-solver.rb

```diff
@@ -15,21 +15,20 @@ class CeresSolver < Formula
     sha256 "0a091d6adf630d059340d1c4e69836fc9ecbbac804b23f855095ad9b0473a6b0" => :mojave
   end
 
-  depends_on "cmake"
+  depends_on "cmake" => [:build, :test]
   depends_on "eigen"
   depends_on "gflags"
   depends_on "glog"
   depends_on "metis"
+  depends_on "openblas"
   depends_on "suite-sparse"
+  depends_on "tbb"
```

Comment:
@SeekingMeaning, I just noticed you added `metis` again and added `openblas` and `tbb`. I think those three are only indirect dependencies and pulled in via `suite-sparse`, and not directly by ceres, which is why I thought we didn't need them here. If having the indirect dependencies is what we want, then ignore my comment. (TBB used to be a direct dependency in 1.14, but they removed it in 2.0.)

See also http://ceres-solver.org/installation.html

```
brew deps --1 suite-sparse
metis
openblas
tbb
```

Reply:
Yeah I know; we should add them as dependencies anyways since ceres-solver links to them